data/web/inc/prerequisites.inc.php

```diff
@@ -5,8 +5,6 @@
   header('Location: /SOGo', true, 307);
   exit;
 }*/
-// Load core functions first
-require_once $_SERVER['DOCUMENT_ROOT'] . '/inc/functions.inc.php';
```

Comment:
Why did you decide to move that in the first place, @feldsam?

Reply:
because function from that file is used in vars.inc.php